Makefile

```diff
@@ -164,23 +164,27 @@ localnet_up: ## Starts localnet
 .PHONY: localnet_down
 localnet_down: ## Delete resources created by localnet
 	tilt down
-	kubectl delete secret celestia-secret || exit 1
 
 .PHONY: localnet_regenesis
 localnet_regenesis: ## Regenerate the localnet genesis file
 # NOTE: intentionally not using --home <dir> flag to avoid overwriting the test keyring
-	ignite chain init
-	mkdir -p $(POKTROLLD_HOME)/config/
-	cp -r ${HOME}/.poktroll/keyring-test $(POKTROLLD_HOME)
-	cp ${HOME}/.poktroll/config/*_key.json $(POKTROLLD_HOME)/config/
-	cp ${HOME}/.poktroll/config/genesis.json $(POKTROLLD_HOME)/config/
-	ADDRESS=$$(jq -r '.address' $(POKTROLLD_HOME)/config/priv_validator_key.json); \
-	PUB_KEY=$$(jq -r '.pub_key' $(POKTROLLD_HOME)/config/priv_validator_key.json); \
-	# NB: Currently the stake => power calculation is constant; however, cosmos-sdk \
-	# intends to make this parameterizable in the future. \
-	POWER=$$(yq ".validators[0].bonded" ./config.yml | sed 's,000000upokt,,'); \
-	NAME=$$(yq ".validators[0].name" ./config.yml); \
-	jq --argjson pubKey "$$PUB_KEY" '.consensus["validators"]=[{"address": "'$$ADDRESS'", "pub_key": $$pubKey, "power": "'$$POWER'", "name": "'$$NAME'"}]' $(POKTROLLD_HOME)/config/genesis.json > temp.json && mv temp.json $(POKTROLLD_HOME)/config/genesis.json
+# NB: Currently the stake => power calculation is constant; however, cosmos-sdk
+# intends to make this parameterizable in the future.
```

Comment:
For some reason, make quietly ignored everything after the comment section. So I moved them out of code.